test/verify/check-storage-lvm2

```diff
@@ -33,22 +34,21 @@ class TestStorage(StorageCase):
         self.login_and_go("/storage")
         b.wait_in_text("#drives", "VirtIO")
 
-        m.add_disk("50M", serial="DISK1")
-        m.add_disk("50M", serial="DISK2")
-        b.wait_in_text("#drives", "DISK1")
-        b.wait_in_text("#drives", "DISK2")
-
-        dev_1 = "/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1"
-        dev_2 = "/dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK2"
+        dev_1 = self.add_ram_disk()
+        dev_2 = self.add_loopback_disk()
```

Comment:
I think we should use `add_loopback_disk` for both disks, for simplicity.  Or is there a reason that one needs to be a RAM disk, and the other not?

Reply:
Yes, there is. loopback devices are a terrible simulation of real disks. They don't appear in the "Drives" list on the overview page, for example. OTOH `scsi_debug` is a pretty faithful emulation of a real hard disk, so I'd much rather test against that as much as possible. With using both we have more diversity in drive types.

Reply:
Ok, that makes sense, thanks!